lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -260,6 +262,26 @@ void LocalEnforcer::aggregate_records(
   }
   complete_termination_for_released_sessions(
       session_map, sessions_with_reporting_flows, session_update);
+  cleanup_dead_sessions(dead_sessions_to_cleanup);
+}
+
+void LocalEnforcer::cleanup_dead_sessions(
+    std::vector<RuleRecord>& dead_sessions_to_cleanup) {
+  if (dead_sessions_to_cleanup.size() == 0) {
+    return;
+  }
+  for (const RuleRecord& record : dead_sessions_to_cleanup) {
+    MLOG(MINFO) << "Removing all flows for " << record.sid() << ", "
+                << record.ue_ipv4()
+                << " since the UE no longer exists in SessionD";
+    Teids empty_teids;  // Teids are not used in 1.4
```

Comment:
We will have to remember to add the Teids here once they are included on the report

Maybe @koolzz can add the teids to the message so we can start using teids here

Reply:
Yea for v1.4 we have to either have pipelined send teid or do something else :)